classes/monograph/ChapterDAO.inc.php

```diff
@@ -11,7 +11,7 @@
  * @ingroup monograph
  *
  * @see Chapter
- * @see AuthorDAO
+ * @see author\DAO
```

Comment:
I think here you would need `@see \APP\author\DAO`

Reply:
changed